package.json

```diff
@@ -32,7 +32,8 @@
     "react-virtualized": "^9.21.2",
     "react-window": "^1.8.5",
     "redux": "^4.0.5",
-    "typeface-roboto": "0.0.75"
+    "typeface-roboto": "0.0.75",
+    "@deck.gl/extensions": "latest"
```

Comment:
please use a fixed version, and also commit the resulting package-lock.json

Reply:
Looks like we already have @deck.gl/extensions@8.0.17 in master, do you need this ?